### ISSUE TITLE
Can't install eslint-nibble without `git` on system

### ISSUE THREAD

**Issue by probil** [IanVS/eslint-nibble#62]
Throws an error when trying to install:

```
/usr/app # npm install --no-save eslint-nibble
npm WARN deprecated joi@14.3.1: This module has moved and is now available at @hapi/joi. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm ERR! path git
npm ERR! code ENOENT
npm ERR! errno ENOENT
npm ERR! syscall spawn git
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/ianvs/eslint-stats.git
npm ERR! enoent 
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-01-28T11_52_03_443Z-debug.log

```
---
Here is my env:
```
node: v10.16.3
os: linux alpine
npm: v6.9.0
eslint: v6.1.0

**Comment by ljharb** [comment]
Looks like you're missing `git` on your alpine system. Either way this would be better filed on npm than here.

**Comment by IanVS** [comment]
Yeah, sorry, I don't think this has anything to do with eslint-nibble.  Make sure you've got `git` installed, and let me know if you have any other trouble.

**Comment by probil** [comment]
I don't have any problems with a bunch of other tools I use.

It seems like the issue is directly connected to this dependency:
https://github.com/IanVS/eslint-nibble/blob/d9edd1f901c3f04a552cdc333c262e06e9e70d6b/package.json#L42

> I don't think this has anything to do with eslint-nibble. 

Hosting your forked `eslint-stats` on npm as a separate package would fix the issue. I mean something under your namespace, e.g. `@ianvs/eslint-stats` 🤔

**Comment by IanVS** [comment]
Hm, I see.  So you don't want `git` installed? And npm is trying to use it to install my fork from github.  Pointing to a git commit is valid in a `package.json` (https://docs.npmjs.com/files/package.json#github-urls), so this kind of does seem like an issue in `npm`.

That said, if I get some time, I'll try to do as you suggest and publish to a namespace.

**Comment by ljharb** [comment]
npm supports installing from git by default, so this is an issue with your system - anything with node and npm also needs git to be fully functional.

**Comment by indvinoth** [comment]
> Hm, I see. So you don't want `git` installed? And npm is trying to use it to install my fork from github. Pointing to a git commit is valid in a `package.json` (https://docs.npmjs.com/files/package.json#github-urls), so this kind of does seem like an issue in `npm`.
> 
> That said, if I get some time, I'll try to do as you suggest and publish to a namespace.

I am facing a different issue with the `eslint-stats` package, When I install packages through the internet I'm not facing any issues but  if I try to install behind the corporate firewall with privately hosted artifactory this fails. There is no connectivity to github and there is prompt asking to choose a version. This was working fine earlier and started to fail from last week.

**Comment by IanVS** [comment]
@indvinoth I haven't changed anything on this end, so I have to imagine that either something in npm has changed, or something in your corporate firewall rules.  I'm sorry, but I don't think I can help with that one.  

But I have published eslint-stats as @ianvs/eslint-stats, and released [eslint-nibble@6.0.0-beta.1](https://github.com/IanVS/eslint-nibble/releases/tag/v6.0.0-beta.1).

**Comment by indvinoth** [comment]
@IanVS Thanks for your response. I was able to sort it out with our internal Artifactory
